### PR TITLE
chore(EMS-3077): No PDF - Policy - Modular UI routes

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
Since No PDF has grown , we have added a lot of policy routes.

This PR splits up all "policy" UI routes and constants so that they are easier to read and maintain.

## Resolution :heavy_check_mark:
- Create a directory for each child policy route's constants.
- Create a directory for each child policy UI route.

## Miscellaneous :heavy_plus_sign:
- Update `test.yml` to include a missing E2E test.

### CODE REVIEW

File: src/ui/server/constants/routes/insurance/policy/another-company.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import POLICY_ROOT from './root';
+
+const ROOT = `${POLICY_ROOT}/another-company`;
+
+const ANOTHER_COMPANY_ROUTES = {
+  ANOTHER_COMPANY: ROOT,
+  ANOTHER_COMPANY_SAVE_AND_BACK: `${ROOT}/save-and-go-back`,
+  OTHER_COMPANY_DETAILS: `${POLICY_ROOT}/other-company-details`,
```
Comment (Zainzzkk) [inline src/ui/server/constants/routes/insurance/policy/another-company.ts:8]: could this have `OTHER_COMPANY_DETAILS` root?

File: src/ui/server/constants/routes/insurance/policy/multiple-contract-policy.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+const ROOT = `${POLICY_ROOT}/multiple-contract-policy`;
+
+const MULTIPLE_CONTRACT_POLICY_ROUTES = {
+  MULTIPLE_CONTRACT_POLICY: ROOT,
+  MULTIPLE_CONTRACT_POLICY_SAVE_AND_BACK: `${ROOT}/save-and-go-back`,
+  MULTIPLE_CONTRACT_POLICY_CHANGE: `${ROOT}/change`,
+  MULTIPLE_CONTRACT_POLICY_CHECK_AND_CHANGE: `${ROOT}/check-and-change`,
+  MULTIPLE_CONTRACT_POLICY_EXPORT_VALUE: `${ROOT}/export-value`,
+  MULTIPLE_CONTRACT_POLICY_EXPORT_VALUE_SAVE_AND_BACK: `${ROOT}/export-value/save-and-go-back`,
```
Comment (Zainzzkk) [inline src/ui/server/constants/routes/insurance/policy/multiple-contract-policy.ts:11]: const for EXPORT_VALUE?

File: src/ui/server/constants/routes/insurance/policy/single-contract-policy.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import POLICY_ROOT from './root';
+
+const ROOT = `${POLICY_ROOT}/single-contract-policy`;
+
+const SINGLE_CONTRACT_POLICY_ROUTES = {
+  SINGLE_CONTRACT_POLICY: ROOT,
+  SINGLE_CONTRACT_POLICY_SAVE_AND_BACK: `${ROOT}/save-and-go-back`,
+  SINGLE_CONTRACT_POLICY_CHANGE: `${ROOT}/change`,
+  SINGLE_CONTRACT_POLICY_CHECK_AND_CHANGE: `${ROOT}/check-and-change`,
+  SINGLE_CONTRACT_POLICY_TOTAL_CONTRACT_VALUE: `${ROOT}/total-contract-value`,
```
Comment (Zainzzkk) [inline src/ui/server/constants/routes/insurance/policy/single-contract-policy.ts:10]: const for TOTAL_CONTRACT_VALUE?

File: e2e-tests/constants/routes/insurance/policy/another-company.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import POLICY_ROOT from './root';
+
+const ROOT = `${POLICY_ROOT}/another-company`;
+
+const ANOTHER_COMPANY_ROUTES = {
+  ANOTHER_COMPANY: ROOT,
+  ANOTHER_COMPANY_SAVE_AND_BACK: `${ROOT}/save-and-go-back`,
+  OTHER_COMPANY_DETAILS: `${POLICY_ROOT}/other-company-details`,
```
Comment (Zainzzkk) [inline e2e-tests/constants/routes/insurance/policy/another-company.js:8]: const for OTHER_COMPANY_DETAILS?

File: e2e-tests/constants/routes/insurance/policy/multiple-contract-policy.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import POLICY_ROOT from './root';
+
+const ROOT = `${POLICY_ROOT}/multiple-contract-policy`;
+
+const MULTIPLE_CONTRACT_POLICY_ROUTES = {
+  MULTIPLE_CONTRACT_POLICY: ROOT,
+  MULTIPLE_CONTRACT_POLICY_SAVE_AND_BACK: `${ROOT}/save-and-go-back`,
+  MULTIPLE_CONTRACT_POLICY_CHANGE: `${ROOT}/change`,
+  MULTIPLE_CONTRACT_POLICY_CHECK_AND_CHANGE: `${ROOT}/check-and-change`,
+  MULTIPLE_CONTRACT_POLICY_EXPORT_VALUE: `${ROOT}/export-value`,
```
Comment (Zainzzkk) [inline e2e-tests/constants/routes/insurance/policy/multiple-contract-policy.js:10]: const for EXPORT_VALUE?

File: src/ui/server/constants/routes/insurance/policy/loss-payee.ts [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+const LOSS_PAYEE_ROUTES = {
+  LOSS_PAYEE_ROOT: ROOT,
+  LOSS_PAYEE_SAVE_AND_BACK: `${ROOT}/save-and-back`,
+  LOSS_PAYEE_CHANGE: `${ROOT}/change`,
+  LOSS_PAYEE_CHECK_AND_CHANGE: `${ROOT}/check-and-change`,
+  LOSS_PAYEE_DETAILS_ROOT,
+  LOSS_PAYEE_DETAILS_CHANGE: `${LOSS_PAYEE_DETAILS_ROOT}/change`,
+  LOSS_PAYEE_DETAILS_CHECK_AND_CHANGE: `${LOSS_PAYEE_DETAILS_ROOT}/check-and-change`,
+  // TODO: remove ROOT from everything apart from the the root
```
Comment (Zainzzkk) [inline src/ui/server/constants/routes/insurance/policy/loss-payee.ts:16]: should these TODOs be addressed in this PR or a later one?

Comment (ttbarnes) [reply src/ui/server/constants/routes/insurance/policy/loss-payee.ts:16]: This will be in the next PR, half complete locally. Will change a lot of files.

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**